contrib/kind-common

```diff
@@ -576,3 +582,32 @@ kubectl_wait_dnsnameresolver_pods() {
   echo "Waiting for pods in dnsnameresolver-operator namespace to become ready (timeout ${timeout})..."
   kubectl wait -n dnsnameresolver-operator --for=condition=ready pods --all --timeout=${timeout}s
 }
+
+deploy_primary_udn_nad() {
+  cat <<EOF | run_kubectl create -f -
```

Comment:
please make sure create (or any of the changes) doesnt fail on redeploy
(this is create that might fail if already exists iiuc)
`./kind.sh -mne --deploy` which reuse the kind cluster but redeploy everything else

Reply:
that could be it. I couldn't figure out why we start failing on other unrelated lanes, but if kind does this kind of thing then it could explain it. Let's change to apply and see.

Reply:
`--deploy` is not used by anyone beside local
apply does fix it (tried locally)
I am not sure about the other lanes at all, but i bet it wont be because of this change specifically (apply / create)

Reply:
This script contains other create commands, so I'm not sure. 
But it doesn't hurt to try.